apps/storefront/src/Layout/MainMenu.tsx

```diff
@@ -39,7 +39,7 @@ const MainMenu: FC<MainMenuProps> = ({ loginDisclosure }) => {
   const megaMenuDisclosure = useDisclosure();
   const [catalogs, setCatalogs] = useState<Catalog[]>([]);
   const [selectedCatalog, setSelectedCatalog] = useState<string>("");
-  const [lineItems, setLineItems] = useState<LineItem[]>();
+  const [, setLineItems] = useState<LineItem[]>();
```

Comment:
can we remove this if we aren't using the lineitems for anything?

Reply:
oops yep